docs/sanic/response.md

```diff
@@ -0,0 +1,99 @@
+# Response
+
+You can use functions in `sanic.response` module to create response.
```

Comment:
Mostly it's just the response vs responses: "You can use functions in `sanic.response` module to create responses" would be great too.